benchmark/buffers/buffer-swap.js

```diff
@@ -5,8 +5,8 @@ const common = require('../common.js');
 const bench = common.createBenchmark(main, {
   aligned: ['true', 'false'],
   method: ['swap16', 'swap32', 'swap64'/* , 'htons', 'htonl', 'htonll' */],
-  len: [8, 64, 128, 256, 512, 768, 1024, 1536, 2056, 4096, 8192],
-  n: [5e7]
+  len: [64, 256, 768, 1024, 2056, 8192],
+  n: [1e6]
```

Comment:
This benchmark is currently likely the worst of all.

I removed half of the sizes since the length changes result in a curve. We could likely even remove more length variations.

The iteration count was significantly lowered to get this benchmark to a reasonable runtime. 

In total this should reduce the runtime by 30x and I hope this is enough but it'll likely still be more than it should ideally have.